@here/harp-omv-datasource/lib/OmvDecoder.ts

```diff
@@ -70,11 +70,7 @@ export class Ring {
         readonly contourOutlines?: boolean[],
         readonly points: number[] = contour
     ) {
-        this.isOuterRing = this.area() < 0;
-    }
-
-    get isInnerRing(): boolean {
-        return !this.isOuterRing;
+        this.winding = Math.sign(this.area());
```

Comment:
What about 0? The winding will be 0 as well. Not sure if the code will handle it properly.

Reply:
that should not really happen, unless the data is broken.

Reply:
i changed the condition to cover this case